### ISSUE TITLE
Level: One from each side

### ISSUE THREAD

**Issue by andybalaam** [andybalaam/rabbit-escape#519]
```
 Q/t  \Q 
   ###   
      /  
M /OOO/ M
MMM###MMM
:author_name=Andy Balaam
:author_url=http://artificialworlds.net/rabbit-escape/
:block=2
:bridge=2
:brolly=2
:description=So many exits, but using them might be tricky...
:dig=2
:explode=2
:name=One from each side
:num_rabbits=2
```

See https://github.com/andybalaam/rabbit-escape/blob/master/rabbit-escape-engine/src/rabbitescape/levels/staging/rabbots/One-from-each-side.rel for hints and solution.

Intended for #520 Rabbots

**Comment by tttppp** [comment]
I enjoyed this.

```
:j:p<s9R:^$1R:L:a1/vj:N:R:/:-Uv:j:QU<ja.v:9#<:R.s-N^/:/:vs19jv:0vQR:UUv^1/8:
```

**Comment by andybalaam** [comment]
```
$ ./runrabbit --decode ':j:p<s9R:^$1R:L:a1/vj:N:R:/:-Uv:j:QU<ja.v:9#<:R.s-N^/:/:vs19jv:0vQR:UUv^1/8:'
\BlirByXB%b{;hB\B!7)yBcB%ByGrc!B7yuBy!BX!)iBBuB6{bl=y=B!X))u=BBBu{ybM7l7b)u4

```

Am I missing something?

**Comment by andybalaam** [comment]
Btw my solution is:
```
2NdBd6feBX@W^@d-'1I9,QdBdNI0,^1]d/dd@L'pd_-1
```

**Comment by tttppp** [comment]
I must have copied wrong. In words I was doing this:

```
BV#j3B"#:B"#d:d'n#kVB"BBfB'V#o!3oB=3BddhBodhB#rVn3*!V#RRRBGoR#B#RB32*f3(13GnB3#R#']rN*"GGBGBk!o3rrBGf3BBkdVGV3hh(#rGBod:rBkB23G3h##GBBBh#3khd3'B''3d3r3VfRBrrVV239VB9
```

**Comment by andybalaam** [comment]
I am also failing to decode that.

**Comment by tttppp** [comment]
That definitely decodes - although it has apostrophes in it:

Use dig token for rabbot to drop down and another for him to clear slope on right. Either explode him or turn him around using blocker/floater combo on right rabbit.

**Comment by andybalaam** [comment]
I managed to get it to decode by putting it into a file called tmp.txt and then doing:

```
./runrabbit --decode $(cat tmp.txt)
```

**Comment by andybalaam** [comment]
No matter what I tried quoting it directly I failed.

**Comment by andybalaam** [comment]
Released in 0.11

**Comment by colonelfazackerley** [comment]
slightly different at the end
pl#(T{z5(TT5S|Q5l4(T(Q5* 5 y5 {:Sz|5U54-Su=* 5Up5%S(S5%z{ T5T
